### DISCUSSION TITLE
Q-uploader - Only send headers, without the images

### DISCUSSION THREAD

**Discussion by davidwbsc** [quasarframework/quasar#13344] [CLI - PWA mode] [answered]
Is it possible to disable the sending of images with the q-uploader?

The reason being, I am converting my images to base64 and then sending them as headers, so I don't need to send the images too.

**Comment by metalsadman** [comment, marked as answer]
use other way of uploading (axios/fetch etc...) not the one within its api.

**Reply by davidwbsc** [reply]
Thanks for your reply.

So there wouldn't be a way to do it using the existing uploader UI?

**Reply by metalsadman** [reply]
you can still use it, just use `header` slot so you can change how it would behave when you click on upload, check this example https://quasar.dev/vue-components/uploader#example--custom-header
ie. in your upload button you can change the `@click` handler to call your custom upload api call.

**Reply by davidwbsc** [reply]
Ah, thanks, yeah, that's doing the job for me!
